app/src/main/java/com/czbix/xposed/wifipassword/ServerPatch.java

```diff
@@ -60,16 +60,6 @@ protected void afterHookedMethod(MethodHookParam param) throws Throwable {
 
     // copied from com.android.server.wifi.WifiConfigStore#readNetworkVariables
     private static String convertSsid(ClassLoader classLoader, String ssid) {
-        if (!TextUtils.isEmpty(ssid)) {
```

Comment:
I (incorrectly) assumed this was used to add quotation marks to the SSID when copied to clipboard. I've gone ahead and removed this change from the commit, so it should be safe to merge now.